### PR TITLE
feat: use data resource settings for toggling url links/buttons

### PR DESCRIPTION
**Pull request by donni106**
SVA-764

|without setting|with setting|
|---|---|
|Simulator Screen Shot - iPhone 12 mini - 2022-11-29 at 17 41 28|Simulator Screen Shot - iPhone 12 mini - 2022-11-29 at 17 41 40|

### CODE REVIEW

File: src/components/screens/EventRecord.js [hunk 1]
```diff
@@ -182,6 +184,22 @@ export const EventRecord = ({ data, route }) => {
           title={texts.eventRecord.operatingCompany}
         />
 
+        {settings?.displayOnlySummary === 'true' && !!settings?.onlySummaryLinkText && (
```
Comment (ardasnturk) [inline src/components/screens/EventRecord.js:187]: Is it possible to replace "true" with another variable? Why is the string value coming?

File: src/components/screens/EventRecord.js [hunk 2]
```diff
@@ -182,6 +184,22 @@ export const EventRecord = ({ data, route }) => {
...
+              const { url } = item;
+
+              return (
+                <Button
+                  key={index}
+                  title={settings.onlySummaryLinkText}
+                  onPress={() => openLink(url, openWebScreen)}
+                />
+              );
+            })}
```
Comment (ardasnturk) [inline src/components/screens/EventRecord.js:199]: Here we only use the `url` information from the item and there is no control inside the `Button` component. How about using it as `webUrls.map(({ url }, index) => {...}` ?
for example:

```
{webUrls.map(({ url }, index) => (
  <Button
    key={index}
    title={settings.onlySummaryLinkText}
    onPress={() => openLink(url, openWebScreen)}
  />
))}
```

**Review by ardasnturk — COMMENTED** [review]
Please give me more information so I can try.

File: src/components/screens/EventRecord.js (hunk 1 above)
Comment (donni106) [reply src/components/screens/EventRecord.js:187]: i don't understand what you mean with replacing true?
`displayOnlySummary` comes as `'true'` or `'false'` per GraphQL and i don't know why it is not boolean.

File: src/components/screens/EventRecord.js (hunk 2 above)
Comment (donni106) [reply src/components/screens/EventRecord.js:199]: good idea! i like the simplicity about this.

File: src/components/screens/EventRecord.js (hunk 1 above)
Comment (ardasnturk) [reply src/components/screens/EventRecord.js:187]: Okay, I got my answer. Only `'true'` or `'false'` string values are coming. 👍

File: src/components/screens/EventRecord.js (hunk 2 above)
Comment (donni106) [reply src/components/screens/EventRecord.js]: 8987358

**Review by ardasnturk — APPROVED** [review]

**Pull request merged**